README.md

```diff
@@ -24,5 +24,10 @@ A UX project to get me up to speed on necessary skills and technologies I will b
 ### Unit 3
 Unit 3 is where the project will diverge a bit from its previous iteration. Because I intend to implement Polymer for the project instead of Angular, the composition of the project is different (i.e., there is a bower_components folder).
 - Because of this, my .gitignore also ignores bower_components.
+  #### Standards Implemented
+  - Polymer dependency
+  - Dev dependencies
+  - Gulpfile with LiveReload
   #### Struggles
   - Since we no longer use Angular, I wanted to install Polymer as a dependency instead of Angular. This came with a host of complications. I spent a lot of time trying to find a package on npm or even Yarn that would work, but I ended up just installing the Bower package.
+  - (3.2) Not sure yet what to do with two disparate package files.
```

Comment:
Shouldn't need to worry about it now 😎 

However, most of our projects use both npm and bower. It really just comes down to making sure you're using the correct library from the right directory (i.e. `node_modules` or `bower_components`).